lib/authenticate_methods.rb

```diff
@@ -34,14 +34,14 @@ def authenticate_admin!
 
   protected
 
-
-  # When the external site provides secure params the're
+  # When the external site provides signed params they're
   # requesting the person either be
-  # (1) automatically logged in if their UUID is known to us
-  # (2) Their UUID is remembered after they login so they're automatically logged in on subsequent visits
-  # (3) directed through a partially pre-populated sign up process, with the UUID being remembered at the end.
+  # (1) automatically logged in if their UUID is known to us, or
+  # (2) allowed to log in or sign up where we pre-populate some fields using
+  #     the signed data and remember their external UUID so that future requests
+  #     with signed params with this UUID can be automatically logged in
```

Comment:
I'm glad you noted the (2) option, but since the implementation was `something || another_thing` I changed the comment to just have two points too.